spec/cancan/matchers_spec.rb

```diff
@@ -1,29 +1,57 @@
 require "spec_helper"
 
 describe "be_able_to" do
-  it "delegates to can?" do
-    expect(object = double).to receive(:can?).with(:read, 123) { true }
-    expect(object).to be_able_to(:read, 123)
-  end
+  let(:object) { double }
```

Comment:
since is the subject of our test we can use the `subject` syntax and simplify our tests.
`subject { double }`